ssh-tunnel

```diff
@@ -2,6 +2,9 @@
 #
 # Creates an SSH Tunnel to the specified server.
 
+reset='\033[0m'       # Text Reset
+cyan='\033[0;36m'         # Cyan
```

Comment:
Gorgeous color

Reply:
Hahaha, yep, it's pretty nice :) My feeling is info=cyan, warning=yellow, error=red.

Although people with a white terminal background (do they even exist???) might disagree with me, since cyan is such a bright colour.

Reply:
Hmm, white terminal background... they must have super vision.